src/Stryker.Core/Stryker.Core/Baseline/Providers/AzureFileShareBaselineProvider.cs

```diff
@@ -14,16 +14,25 @@ namespace Stryker.Core.Baseline.Providers
 {
     public class AzureFileShareBaselineProvider : IBaselineProvider
     {
+        private const string DefaultOutputDirectoryName = "StrykerOutput";
+        private const string BaselinesDirectoryName = "Baselines";
+
         private readonly IStrykerOptions _options;
         private readonly HttpClient _httpClient;
         private readonly ILogger<AzureFileShareBaselineProvider> _logger;
-        private const string _outputPath = "StrykerOutput/Baselines";
+        private readonly string _outputPath;
 
         public AzureFileShareBaselineProvider(IStrykerOptions options, HttpClient httpClient = null)
         {
             _options = options;
             _httpClient = httpClient ?? new HttpClient();
             _logger = ApplicationLogging.LoggerFactory.CreateLogger<AzureFileShareBaselineProvider>();
+
+            _outputPath = $"{DefaultOutputDirectoryName}/{BaselinesDirectoryName}";
+            if (!string.IsNullOrWhiteSpace(options.ProjectName))
+            {
+                _outputPath = $"{options.ProjectName}/{BaselinesDirectoryName}";
```

Comment:
I'm not sure about using ProjectName for this. That option already has a purpose in Stryker. If someone needs that option to run Stryker on his project and doesn't need a sub folder in azure fileshare it might be confusing. @rouke-broersma what about creating a new option for this? (we could do that in a seperate PR on the V1 branch after the options rework)

Reply:
I dislike having so many different options, this is why I already moved projectname to a more general project-info.name in the new options instead of dashboard.name.

Options should be generally applicable to promote reuse imo.